ccd-definition/CaseEventToFields/CareSupervision/openCase.json

```diff
@@ -1,12 +1,37 @@
 [
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "openCase",
+    "CaseFieldID": "migrateToMoWarning",
+    "PageFieldDisplayOrder": 1,
+    "DisplayContext": "READONLY",
+    "PageID": 1,
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "PageShowCondition": "showManageOrgWarning=\"YES\""
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "openCase",
+    "CaseFieldID": "showManageOrgWarning",
+    "PageFieldDisplayOrder": 1,
+    "DisplayContext": "READONLY",
+    "PageID": 2,
```

Comment:
Is this supposed to change? Should the `PageFieldDisplayOrder` be 2? And the `PageDisplayOrder` as well. Same for the name field below